src/test/scala/CoreServerTests.scala

```diff
@@ -205,9 +197,9 @@ class CoreServerTest extends WordSpec with Matchers with ScalatestRouteTest {
             case Success(msgs) =>
               msgs.last match {
                 case m: OverallSuccessMessage =>
-                  assert(f != verificationErrorFile)
+                  assert(f != verificationError_file)
```

Comment:
Te notation is a bit weird. Why not use camelCase for all variable names?

Reply:
But since this is a test, feel free to choose any notation you want (as long as it's consistent).